### PR TITLE
PB3 Entering new clients

### PR DESCRIPTION
**Pull request by caitlinbulloch**
This was created to allow for new client's and their information to be added to the client table.

### CODE REVIEW

**Review by amchugh689 — APPROVED** [review]
this looks good! :)

File: sql/Product-Backlog-3.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+CREATE TABLE Client (
+  ClientID int AUTO_INCREMENT PRIMARY KEY,
+    Forename varchar(20) NOT NULL,
+    Surname varchar(20) NOT NULL,
+    Address varchar(50) NOT NULL,
+    Phone varchar(15) NOT NULL
+);
+
+INSERT INTO Client (Forename, Surname, Address, Phone)
+VALUES ('Nicole','Brown','12 Drumstick lane','174882123');
```
Comment (jkelso662) [inline sql/Product-Backlog-3.sql]: Weird phone number! Also could probably do with a few extra rows!

File: sql/Product-Backlog-3.sql [hunk 2]
```diff
@@ -0,0 +1,12 @@
+USE db_PICASO_OdhranH;
+
+CREATE TABLE Client (
+  ClientID int AUTO_INCREMENT PRIMARY KEY,
+    Forename varchar(20) NOT NULL,
```
Comment (jkelso662) [inline sql/Product-Backlog-3.sql]: Inconsistent tabs

File: sql/Product-Backlog-3.sql [hunk 3]
```diff
@@ -0,0 +1,12 @@
+USE db_PICASO_OdhranH;
```
Comment (jkelso662) [inline sql/Product-Backlog-3.sql]: Is this the best name for this file?

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Couple of things, mostly just check your test data, tabs, and double check file naming. Make sure you add a MR description so that the customer knows what feature this is for!

**Pull request merged**